examples/StandardFirmataWiFi/wifiConfig.h

```diff
@@ -44,46 +45,57 @@ WiFiStream stream;
  */
 //#define WIFI_101
 
-//do not modify these next 4 lines
+//do not modify these next 5 lines
 #ifdef WIFI_101
+#include <WiFi101.h>
 #include "utility/WiFi101Stream.h"
-WiFi101Stream stream;
+WiFiStream stream;
 #endif
 
 /*
- * OPTION C: Configure for HUZZAH
+ * OPTION C: Configure for ESP8266
+ *
+ * This will configure StandardFirmataWiFi to use the ESP8266WiFi library for boards
+ * with an ESP8266 chip. It is compatible with 802.11 B/G/N networks.
+ *
+ * To enable, uncomment the #define ESP8266_WIFI below and verify the #define values under
+ * options A and B are commented out.
  *
- * HUZZAH is not yet supported, this will be added in a later revision to StandardFirmataWiFi
+ * IMPORTANT: You must have the esp8266 board support installed. To easily install this board, open the board manager via:
+ * Arduino IDE Menus: Tools > Board > Manage Boards > filter search for "esp8266" > Select the result and click 'install'
  */
+//#define ESP8266_WIFI
 
-//------------------------------
-// TODO
-//------------------------------
-//#define HUZZAH_WIFI
+//do not modify these next 5 lines
+#ifdef ESP8266_WIFI
+#include <ESP8266WiFi.h>
+#include "utility/WiFiStream.h"
+WiFiStream stream;
+#endif
 
 /*
- * OPTION D: Configure for ESP6288
+ * OPTION D: Configure for HUZZAH
  *
- * ESP6288 is supported through its Arduino Core at
- * https://github.com/esp8266/Arduino/
+ * HUZZAH with CC3000 is not yet supported, this will be added in a later revision to StandardFirmataWiFi.
+ * For HUZZAH with ESP8266 use ESP8266_WIFI.
  */
 
-//#define ESP_WIFI
+//------------------------------
+// TODO
+//------------------------------
+//#define HUZZAH_WIFI
 
-//do not modify these next 4 lines
-#ifdef ESP_WIFI
-#include "utility/ESPWiFiStream.h"
-WiFiStream stream;
-#endif
 
 // STEP 2 [REQUIRED for all boards and shields]
 // replace this with your wireless network SSID
 char ssid[] = "your_network_name";
 
 // STEP 3 [OPTIONAL for all boards and shields]
 // if you want to use a static IP (v4) address, uncomment the line below. You can also change the IP.
-// if this line is commented out, the WiFi shield will attempt to get an IP from the DHCP server
-// #define STATIC_IP_ADDRESS 192,168,1,113
+// if the first line is commented out, the WiFi shield will attempt to get an IP from the DHCP server
+#define STATIC_IP_ADDRESS  192,168,1,113
+#define SUBNET_MASK        255,255,255,0 // REQUIRED for ESP8266_WIFI, ignored for others
```

Comment:
Interestingly the WiFi and WiFi101 support local IP mode without the need to define netmask and gateway but they also support these parameters. From the network point of view the full set is preferable.